pyro/distributions/torch.py

```diff
@@ -287,6 +287,15 @@ def expand(self, batch_shape):
         return super(TransformedDistribution, self).expand(batch_shape)
 
 
+class TransformModule(torch.distributions.Transform, torch.nn.Module):
+    def __init__(self):
+        torch.distributions.Transform.__init__(self)
+        torch.nn.Module.__init__(self)
+
+    def __hash__(self):
+        return super(torch.nn.Module, self).__hash__()
```

Comment:
nit: I think it will be simpler to just use `hash(self)`.

Reply:
This won't work because you need to call the `.__hash__()` from the parent class. Doing `.hash(self)` enters an infinite recursion. Is there a neater way to call the parent class method?

Reply:
Sorry, what you have is correct! I was confused - hash(self) would just call `__hash__` method (as it should, I thought it would use `object.__hash__`) itself and go into infinite recursion.